hs_tools_resource/models.py

```diff
@@ -35,190 +33,112 @@ def metadata(self):
         md = ToolMetaData()
         return self._get_metadata(md)
 
-
 processor_for(ToolResource)(resource_processor)
 
 
 class RequestUrlBase(AbstractMetaDataElement):
-    term = 'Request Url Base'
-    value = models.CharField(null=True, max_length="500") # whatever the user gives us- format is "http://www.example.com/{resource-info}"
-
+    term = 'RequestUrlBase'
+    value = models.CharField(max_length=1024, null=True)
+    resShortID = models.CharField(max_length=128, default="UNKNOWN")
```

Comment:
@pkdash @selimnairb On a resource landing page the goal is to determine if this specific resource type has a supported web app resource associated with it. However, we must check the permissions of the associated web app resource object to verify if it can show up on the current resource's landing page. Storing the resource short id of the web app in its metadata is the work-around to do this. If there is another method that has been developed, we will gladly implement it.